### PR TITLE
Metrics lint

### PR DESCRIPTION
**Pull request by manishtomar**
flake8 fixes for `metrics.py` and `test_metrics.py`

### CODE REVIEW

File: otter/metrics.py [hunk 1]
```diff
@@ -180,35 +191,40 @@ def get_all_metrics(cass_groups, authenticator, service_mapping, region,
...
-    :param str identity_url: URL of identity API to authenticate users given in `conf`
+    :param dict conf: Metrics configuration, will contain tenant ID of tenant
+                      used to ingest metrics and other conf like ttl
     :param str region: which region's metric is collected
-    :param int total_desired: Total number of servers currently desired in a region
-    :param int total_actual: Total number of servers currently there in a region
-    :param int total_pending: Total number of servers currently building in a region
-    :param _treq: Optional treq implementation for testing
+    :param int total_desired: Total number of servers currently desired
+                              in the region
```
Comment (lvh) [inline otter/metrics.py]: Sorry, I may have been unclear about this: the indentation is supposed to be 4, not up until the text.

**Comment by lvh** [comment]
LGTM modulo docstring comments. Sorry about that :( Thanks for working on this!

**Comment by lvh** [comment]
As pointed out by @radix, we temporarily became more lax about unused imports, because we went from full to diff. Therefore, you might also want to run pyflakes otter, although I doubt it matters for this PR, since it's just about lint fixes.

File: otter/metrics.py (hunk 1 above)
Comment (manishtomar) [inline otter/metrics.py]: Why doesn't flake8 capture this? I ran it on the whole file.

Comment (lvh) [inline otter/metrics.py]: flake8 doesn't by default have a docstring linter. pep257 takes care of part of it, but not all of it. sphinx has some plugins, but they're not installed yet.

After those few PRs, we're definitely not done with a fully-functional linter.

**Comment by manishtomar** [comment]
FYI: pyflakes is good with these files.

**Pull request merged**